src/app/story/storyrow/storyrow.component.css

```diff
@@ -17,7 +17,13 @@
     padding-right: 1.5rem;
     position: relative;
     /* Vertically center the text */
-    top: 5px;
+    top: -8px;
+    font-weight: bold;
+}
+
+.node-timestamp {
+    font-size: 10px;
```

Comment:
Could you use `rem` instead of pixels? I try to only use pixels when dealing with a static size like an image.